batch/batch/server/server.py

```diff
@@ -414,7 +434,8 @@ def to_json(self):
                 'Cancelled': state_count.get('Cancelled', 0)
             },
             'exit_codes': {j.id: j.exit_code for j in self.jobs},
-            'attributes': self.attributes
+            'attributes': self.attributes,
+            'is_open': self.is_open
```

Comment:
Is 'is_open' currently consumed by any downstream services, and if not, is the reason for exposing a near-term plan to do so?

Reply:
I needed the API to expose the open/closed status of the batch so that I could [test for it](https://github.com/hail-is/hail/pull/5233/files#diff-ab13c93eb3cd3037ad9eeb3f85824196R42). All the batch tests are written against the HTTP API.